### ISSUE TITLE
please comment on the state of the project in the readme

### ISSUE THREAD

**Issue by mstrhh** [jaked/ocamljs#5]
Is it completely abandoned? Is there a relation to js_of_ocaml?

Just to inform the interested github reader. Thanks.

**Comment by tajmone** [comment]
Yes, it would be nice to be updated on the status of the project, or if there are any active forks around.
